src/dlgAboutDialog.cpp

```diff
@@ -183,6 +254,22 @@ void dlgAboutDialog::setAboutTab(const QString& htmlHead) const
     // clang-format on
 }
 
+QString dlgAboutDialog::createMakerHTML(const aboutMaker maker) const
+{
+    QString coloredText("<span style=\"color:#%1;\">%2</span>");
```

Comment:
I am not sure this helps much - you will need to mark it up as a `QStringLiteral` - to avoid `QT_NO_CAST_FROM_ASCII` issues in the future (if we ever get around to defining it project wide) and from a somewhat biased viewpoint I am not sure that it is too harmful to have similar ~~HTML~~ Qt Rich Text repeated three times...

Reply:
I like the additional layer of indirection for improved readability. Mixing HTML code and C++ code too much seems too confusing. Marked as QStringLiteral nonetheless.